.gitignore

```diff
@@ -1,2 +1,3 @@
 
-public/node_modules/*
\ No newline at end of file
+public/node_modules/*
+config/twitter_oauth.yml
```

Comment:
nvm; token file. should probably use ENV eventually but gitignore-ing the file will do for now